projects/desyn-farm1/index.js

```diff
@@ -1,11 +1,12 @@
 const { getLogs } = require('../helper/cache/getLogs')
 
 module.exports = {
-  methodology: 'On-chain restaking',
+  methodology: 'Product: On-chain restaking<br/>TVL: It includes above<br/>Revenue: Staking rewards, management fees, performance fees',
```

Comment:
I am tired of reverting this every time

Reply:
hi，sorry，guys，I'll pay attention next time.